README.md

```diff
@@ -21,6 +21,19 @@ almost always `'origin/main'` or `'origin/master'`. When `sorald-buildbreaker`
 runs on pull requests, this has the effect of only acting upon violations that
 are contained within the changed lines of the pull request.
 
+### `suggestions-token`
+
+A token to post pull request suggestions with. If provided,
+`sorald-buildbreaker` will use it to post pull request review comments with
+Sorald's repairs as suggestions.
+
+Typically, this should be provided with the `secrets.GITHUB_TOKEN`.
```

Comment:
Maybe you can also mention that the GitHub token is always there in actions and the client does not need to acquire it on their own.

Reply:
Added a link to the GHA auth docs which explain that https://docs.github.com/en/actions/reference/authentication-in-a-workflow